samples/python/getting-started/webfrontend/public/app.js

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+var app = angular.module('myApp', ['ngRoute']);
+
+app.controller('MainController', function($scope, $http) {
+
+    $scope.messages = [];
+    $scope.sayHelloToServer = function() {
+        $http.get("/api?_=" + Date.now()).then(function(response) {
```

Comment:
>Date.now() [](start = 30, length = 10)

Why is this being added as a query param?  It doesn't look like it's being read/included by the backend?